visualstudio/2017/netstandard/NetstandardModuleSample/.nuget/nuget.config

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The idea path should be:
netstandard\
- \windows
   - \vs2017
   - \vs2015
- \linux